### PR TITLE
Add google-apputils

### PR DESCRIPTION
**Pull request by jakirkham**
This should be ready to go just we need to merge this PR first ( https://github.com/conda-forge/staged-recipes/pull/171 ).

### CODE REVIEW

**Comment by jakirkham** [comment]
Passing. :) Any feedback on this one?

File: recipes/google-apputils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - google
+    - google.apputils
+
+  requires:
+    - mox >=0.5
+    - python-dateutil >=1.4
+    - python-gflags >=1.4
+    - pytz >=2010
+    - unittest2 >=0.5.1  # [py26]
```
Comment (ocefpaf) [inline recipes/google-apputils/meta.yaml]: We don't build py26 but it does not hurt to add that.

Comment (jakirkham) [inline recipes/google-apputils/meta.yaml]: Yep, I thought in case someone used this locally.

**Comment by jakirkham** [comment]
Thanks @ocefpaf.

**Pull request merged**